.cbmc-batch/jobs/aws_cryptosdk_decrypt_body/aws_cryptosdk_decrypt_body_harness.c

```diff
@@ -41,7 +41,8 @@ void aws_cryptosdk_decrypt_body_harness() {
 
     struct content_key *content_key;
 
-    uint8_t *tag = can_fail_malloc(props->tag_len);
+    uint8_t *tag = malloc(props->tag_len);
+    // possible that more memory could be allocated, but need to be able to write tag_len bytes to buffer tag.
```

Comment:
Does this PR depends on https://github.com/aws/aws-encryption-sdk-c/pull/566? We should mentioned in the description. All the comments there apply here too.

Reply:
Nope it doesn't! I've re-opened this with some additional improvements in PR #582 where I branch off master. I'll be closing this PR.